models/_sector/dex/trades/base/_schema.yml

```diff
@@ -766,3 +766,22 @@ models:
           seed_file: ref('gridex_base_base_trades_seed')
           filter:
             version: 1
+
+  - name: baso_finance_base_base_trades
+    meta:
+      blockchain: base
+      sector: dex
+      project: baso_finance
+      contributors: sofiat
+    config:
+      tags: [ 'base', 'dex', 'trades', 'baso_finance' ]
+    description: "baso finance base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('baso_finance_base_base_trades_seed')
+          filter:
+            version: 1
```

Comment:
The version seems unmatched to `baso_finance_base.base_trades`, please recheck.

(If the seed file only contains one version, we may remove the filter.)

Reply:
Yeah. It's just one version. I have removed the filter